### ISSUE TITLE
Kyeongki-Do is outdated and should be Gyeonggi-do

### ISSUE THREAD

**Issue by haydonchiugh** [pelias/pelias#967]
Hi Team,

My name is Haydon; I'm a Support Engineer at Greenhouse (greenhouse.com); we recently received the following feedback from a user:

"The user has requested we update the location list on the application form. As per their screenshot, Kyeongki-Do is outdated and should be Gyeonggi-do. Beyond that they just said, "The information is outdated and search performance is truly bad in my point of view.""
![Screenshot 2024-05-02 at 12 23 46 PM](https://github.com/pelias/pelias/assets/95316654/cf4c47b7-e091-499b-8040-a96afd411e8d)

Can you assist in correcting these options?

Thank you in advance for your help.

Haydon

**Comment by alex-habrock-SHG** [comment]
Agreed this is something we are running into with our implementation as well.  Would be nice to get it updated.
